jujugui/static/gui/src/app/components/header-search/header-search.js

```diff
@@ -215,7 +215,7 @@ YUI.add('header-search', function() {
 
     render: function() {
       return (
-        <div className={this._generateClasses()}>
+        <div className={this._generateClasses()} ref="headerSearchContainer">
```

Comment:
The doc does a better job of explaining than I would :)

https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs